### DISCUSSION TITLE
Grouping selected objects?

### DISCUSSION THREAD

**Discussion by RichardScharf** [xournalpp/xournalpp#3920] [Q&A] [answered]
Is there a way to group objects like in SVG editors (e. g. `ctrl+g` in inkscape or excalidraw)?

I find selecting grouped objects a lot easier than manually selecting everything again.

If not, is it possible to create a plugin for it?

**Comment by rolandlo** [comment, marked as answer]
This is related to #1838. It's currently not possible to group objects and with the current plugin API it can't be realized as a plugin. 
Grouping objects may also require changes in the file format.

**Comment by joaohenriquemf** [comment]
I would like to know if there is any provision for this feature. I really like the software because it is easy to draw during the class.

**Reply by Technius** [reply]
This is something that we can consider adding to the new file format. Please open a ticket.

**Reply by RichardScharf** [reply]
That would be absolutely amazing! Thanks for the info👍

Where/how can I open a ticket? I only know how to open Issues.

**Reply by Technius** [reply]
By ticket, I mean issue.

**Comment by bhaveshLathoriya** [comment]
please, set short cut key for hand tool.
i am offen use that tool, i am annoying by clik all time on icon. could you please solve.

**Reply by rolandlo** [reply]
There is already one: Ctrl+Shift+a

**Comment by bhaveshLathoriya** [comment]
Hand tool shortcut?

**Reply by rolandlo** [reply]
Yes
